### PR TITLE
reduce the fear level of the warning of useComputed async

### PR DESCRIPTION
**Pull request by shairez**
# What is it?

- Infra

# Description

When devs see warnings, they aren't used to see the word `Error` in it. 

So I removed it so devs won't think that they're app is broken or something

### CODE REVIEW

File: packages/qwik/src/core/use/use-task.ts [hunk 1]
```diff
@@ -763,7 +763,8 @@ export const runComputed = (
       const stack = new Error(
         'useComputed$: Async functions in computed tasks are deprecated and will stop working in v2. Use useTask$ or useResource$ instead.'
       ).stack;
-      logOnceWarn(stack);
+      const lessScaryStack = stack ? stack.replace(/^Error:\s*/, '') : '';
```
Comment (wmertens) [inline packages/qwik/src/core/use/use-task.ts:766]: if there's no stack it should just return I suppose?

Comment (shairez) [reply packages/qwik/src/core/use/use-task.ts:766]: yeah I guess :)

Comment (shairez) [reply packages/qwik/src/core/use/use-task.ts:766]: although I guess we should just log the warning without the stack... I'll add it

**Review by wmertens — APPROVED** [review]
LGTM!

**Pull request merged**